fe/fe-core/src/main/java/com/starrocks/authentication/AuthenticationManager.java

```diff
@@ -24,8 +33,10 @@ public class AuthenticationManager {
 
     // core data struction
     // user identity -> all the authentication infomation
+    @Expose(serialize = false)
```

Comment:
You may need to invoke `GsonBuilder.excludeFieldsWithoutExposeAnnotation()` in `GsonUtils.java` to make `Expose` annotation to take effect, but this will introduce other problems, maybe not using it and handling this situation manually is a better idea?

Reply:
What are the other problems you're referring to? I'v already added a case to verify the whole process, and it worked fine.

Reply:
Let me make this more clearly, based on the source code comments of `Expose` annotation,
> This annotation has no effect unless you build [Gson](https://www.javadoc.io/static/com.google.code.gson/gson/2.8.9/com.google.gson/com/google/gson/Gson.html) with a [GsonBuilder](https://www.javadoc.io/static/com.google.code.gson/gson/2.8.9/com.google.gson/com/google/gson/GsonBuilder.html) and invoke [GsonBuilder.excludeFieldsWithoutExposeAnnotation()](https://www.javadoc.io/static/com.google.code.gson/gson/2.8.9/com.google.gson/com/google/gson/GsonBuilder.html#excludeFieldsWithoutExposeAnnotation()) method.

but since we use the static object GSON of GsonUtils class to serialize and deserialize object, so if we add `excludeFieldsWithoutExposeAnnotation()` call, this will affect other class who doesn't mark their to be serialized fields with `Expose` annotation.

I am not sure the `@Expose(serialize = false)` works as expected, because i didn't see the corresponding test cases.

Reply:
`testCreateUserPersist` is the case that I was referring to. It persists the `AuthenticationManager` class into stream, and then reloads the image and verifies.

Reply:
Removed `Expose` now.